frontend/src/metabase/dashboard/containers/selectors.jsx

```diff
@@ -0,0 +1,5 @@
+export const getIsBookmarked = (state, props) =>
```

Comment:
There are other selectors in `metabase/dashboard/selectors.js`, I believe it would be better to move this one there